packages/generator-caliper/generators/benchmark/index.js

```diff
@@ -272,6 +272,10 @@ module.exports = class extends Generator {
             answersObject.opts = 'tps: 100, unfinished_per_client: 100';
             this._configWrite();
             break;
+        case 'fixed-load':
+            answersObject.opts = 'startTps:10, transactionLoad:20';
```

Comment:
Could we have a single space added between property and value so that it's consistent with the others, ie
`startTps: 10, transactionLoad: 20`

Reply:
done